DayPicker.js

```diff
@@ -2,14 +2,16 @@
 /* eslint-env node */
 
 var DayPicker = require('./lib/src/DayPicker');
+var DayPickerInput = require('./lib/src/DayPickerInput');
```

Comment:
I share your concern @TrySound 

It could be 'my bad' so here's what I'm doing in my sample [Typescript project](https://github.com/lpcarignan/DayPickerInputTypescript).

I compile Typescript and use Browserify to put everything in a single `index.js` file that I load in the browser. Browserify doesn't include the code in `DayPickerInput.js`. It only includes code from `DayPicker.js`. I can't explain why. I'm not familiar enough with how:

- Typescript transcompile `import { DayPicker, DayPickerInput, DayModifiers } from "react-day-picker";`  to Javascript
- What the `require("react-day-picker")` Javascript statement actually does behind the scenes
- Why only `DayPicker.js` is only loaded by `require("react-day-picker")`

If you hava some insights on this, feel free to share them with me and I can try them in the sample Typescript project.